ui/app/templates/components/secret-create-or-update.hbs

```diff
@@ -29,7 +29,7 @@
         />
       {{/if}}
       {{#if @modelForData.isFolder}}
-        <p class="help is-danger">
+        <p class="help has-text-weight-bold">
```

Comment:
this is-danger here was a very strange class. it was only applying a bold font and nothing else, replaced with a more appropriate helper.